qtpy/__init__.py

```diff
@@ -76,6 +76,34 @@ class PythonQtValueError(ValueError):
     """Error raised if an invalid QT_API is specified."""
 
 
+class QtModuleNotFoundError(ModuleNotFoundError, PythonQtError):
+    """Raised when a Python Qt binding submodule is not installed/supported."""
+    _msg = 'The {name} module was not found for {binding}.'
+    _msg_extra = ''
+
+    def __init__(self, *, name, binding=None, msg=None, **msg_kwargs):
+        self.binding = binding
```

Comment:
```suggestion
    _msg = 'The {name} module was not found{binding}.'
    _msg_binding = ' for {binding}`
    _msg_extra = ''

    def __init__(self, *, name, binding=None, msg=None, **msg_kwargs):
        self.binding = binding
        binding = self._msg_binding.format(binding=binding) if binding else ''
```

Handle binding not being passed.